clients/cobol-lsp-vscode-extension/package.json

```diff
@@ -591,9 +591,11 @@
     "test:integration": "npm run compile && node ./out/test/runTest.js",
     "test:web": "npm run build:web && vscode-test-web --browserType=chromium --headless --extensionDevelopmentPath=. --extensionTestsPath=dist/web/test/suite/extensionTests.js",
     "compile": "tsc -p ./",
-    "lint": "prettier -c .",
-    "pretest": "npm run compile && npm run lint",
-    "pretty": "prettier -w .",
+    "lint-format": "prettier -c .",
+    "lint-quality": "eslint ./src",
+    "lint": "npm run lint-format && npm run lint-quality",
+    "pretest": "npm run compile",
+    "pretty": "eslint --fix ./src && prettier -w .",
```

Comment:
I am not sure about this. Maybe keep the `pretty` command and add a new one `lint-fix` that runs the fixes and prettier after?